packages/@webex/plugin-authorization-browser/test/unit/spec/authorization.js

```diff
@@ -188,11 +187,12 @@ browserOnly(describe)('plugin-authorization-browser', () => {
         });
 
         it('throws a grant error when the url contains one', () => {
-          assert.throws(() => {
-            makeWebexCore(
-              'http://127.0.0.1:8000/?error=invalid_scope&error_description=The%20requested%20scope%20is%20invalid.'
-            );
-          }, /The requested scope is invalid./);
+          try {
+            makeWebexCore('http://127.0.0.1:8000/?error=invalid_scope&error_description=The%20requested%20scope%20is%20invalid.');
```

Comment:
same as other comment - the test will now pass if makeWebexCore doesn't throw at all

Reply:
Changed the syntax.